src/documents/contrast.md

```diff
@@ -0,0 +1,134 @@
+The adjuster presents different forground and background colour themes.
+When the user selects a theme, an [enactor](http://docs.fluidproject.org/infusion/development/Enactors.html)
+(provided by the
+[Infusion Preferences Framework](http://docs.fluidproject.org/infusion/development/PreferencesFramework.html))
+applies the selected theme to the First Discovery Tool interface.
+
+## Using the Contrast Panel
+
+*Option 1*: In the
+[Auxiliary Schema](http://docs.fluidproject.org/infusion/development/AuxiliarySchemaForPreferencesFramework.html),
+specify the name of the panel as the `type` property of the `panel` section:
+```javascript
+"contrast": {
+    "type": "fluid.prefs.contrast",
+    "panel": {
+        "type": "gpii.firstDiscovery.panel.contrast",
+        "container": ".gpiic-fd-prefsEditor-panel-contrast",
+        "classnameMap": {"theme": "@contrast.classes"},
+        "template": "%templatePrefix/contrast.html",
+        "message": "%messagePrefix/contrast.json"
+    }
+}
+```
+
+Working in conjunction with the Auxiliary Schema, the type, its default value and range of
+the text size preference are defined in the
+[Primary Schema](http://docs.fluidproject.org/infusion/development/PrimarySchemaForPreferencesFramework.html):
+```javascript
+fluid.defaults("fluid.prefs.schemas.contrast", {
+    gradeNames: ["fluid.prefs.schemas"],
+    schema: {
+        "fluid.prefs.contrast": {
+            "type": "string",
+            "default": "default",
+            "enum": ["default", "bw", "wb", "by", "yb", "lgdg"]
+        }
+    }
+});
+```
+
+*Option 2*: Outside the context of the First Discovery Tool, developers may wish to create a standalone component:
+```javascript
+var myPanel = gpii.firstDiscovery.panel.contrast(container, options);
+```
+
+## Grades
+
+This component uses the following base
+[grades](http://docs.fluidproject.org/infusion/development/ComponentGrades.html):
+
+* [`gpii.firstDiscovery.attachTooltip.renderer`](attachTooltipRenderer.md)
+* [`fluid.prefs.panel`](http://docs.fluidproject.org/infusion/development/Panels.html)
+
+## Model
+
+This component supports the following
+[model](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ModelComponents.html)
+properties:
+
+| Path   | Description | Values | Default |
+|--------|-------------|--------|---------|
+| `value` | The selected contrast preference | String | undefined |
+
+## Options
+
+This component can be configured using the following
+[options](http://docs.fluidproject.org/infusion/development/ComponentOptionsAndDefaults.html):
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`controlValues`</td>
+        <td>An object that contains a "theme" element that determines values assigned to HTML radio buttons.</td>
+        <td>Object<br/>The "theme" element: array.</td>
```

Comment:
Should be better explained. Probably: An object with a "theme" element that is an array?